dot/network/transaction.go

```diff
@@ -119,36 +120,54 @@ func decodeTransactionHandshake(_ []byte) (Handshake, error) {
 	return &transactionHandshake{}, nil
 }
 
-func (s *Service) createBatchMessageHandler(txnBatch chan *BatchMessage) NotificationsMessageBatchHandler {
-	return func(peer peer.ID, msg NotificationsMessage) (msgs []*BatchMessage, err error) {
+func (s *Service) createBatchMessageHandler(txnBatchCh chan *BatchMessage) NotificationsMessageBatchHandler {
+	go func() {
+		protocolID := s.host.protocolID + transactionsID
+		ticker := time.NewTicker(1 * time.Second)
```

Comment:
it would be nice to configure this using slot duration, i think 1/3 a slot duration is reasonable, that way the node has 1/3 a slot to process the transaction messages and put them in the pool if they want. what do you think?

Reply:
Done